lib/dartdoc.dart

```diff
@@ -128,7 +129,7 @@ class DartDoc {
   List<LibraryElement> _parseLibraries(List<String> files) {
     Set<LibraryElement> libraries = new Set();
     DartSdk sdk = new DirectoryBasedDartSdk(new JavaFile(sdkDir.path));
```

Comment:
If that's the case, we need an `EmbedderSdk` in case there's embedded SDK info...

Maybe better to discuss in person?